doctest/parts/doctest_fwd.h

```diff
@@ -576,12 +578,21 @@ class DOCTEST_INTERFACE String
 
     int compare(const char* other, bool no_case = false) const;
     int compare(const String& other, bool no_case = false) const;
+    const char *compare(const Contains& other) const;
+};
+
+class DOCTEST_INTERFACE Contains {
+public:
+    Contains(const char* string);
+
+    String string;
 };
 
 DOCTEST_INTERFACE String operator+(const String& lhs, const String& rhs);
 
 DOCTEST_INTERFACE bool operator==(const String& lhs, const String& rhs);
 DOCTEST_INTERFACE bool operator!=(const String& lhs, const String& rhs);
+DOCTEST_INTERFACE bool operator!=(const String& lhs, const Contains& rhs);
```

Comment:
Should have `operator==` as well. Just for general utility and maintaining symmetry.